src/lib/Components/Inbox/Conversations/Message.tsx

```diff
@@ -47,29 +48,43 @@ const ArtworkPreviewContainer = styled.View`
   marginBottom: 10
 `
 
-interface Props {
-  message: {
-    senderName: string
-    time: string
-    body: string
-  }
+interface Props extends RelayProps {
+  senderName: string
   artworkPreview?: JSX.Element
 }
 
-export default class Message extends React.Component<Props, any> {
+export class Message extends React.Component<Props, any> {
   render() {
     return (
       <Container>
         <Avatar />
         <TextContainer>
           <Header>
-            <SenderName>{this.props.message.senderName}</SenderName>
-            <MetadataText>{this.props.message.time}</MetadataText>
+            <SenderName>{this.props.senderName}</SenderName>
+            <MetadataText>{this.props.message.created_at}</MetadataText>
           </Header>
           {this.props.artworkPreview && <ArtworkPreviewContainer>{this.props.artworkPreview}</ArtworkPreviewContainer>}
-          <BodyText>{this.props.message.body.split("\n\nAbout")[0]}</BodyText>
+          <BodyText>{this.props.message.raw_text.split("\n\nAbout")[0]}</BodyText>
         </TextContainer>
       </Container>
     )
   }
 }
+
+export default Relay.createContainer(Message, {
```

Comment:
Basically I'm stuck here.

What seems to be happening is the query being generated seems correct, based on the remote JS debugging and looking at the Metaphysics query....but then all the props and everything are just `null`.